go-controller/pkg/ovn/controller/apbroute/network_client.go

```diff
@@ -117,6 +129,18 @@ func (nb *northBoundClient) deleteGatewayIPs(namespace string, toBeDeletedGWIPs,
 			routeInfo.Unlock()
 			continue
 		}
+		pod, err := nb.podLister.Pods(routeInfo.PodName.Namespace).Get(routeInfo.PodName.Name)
+		if err != nil {
```

Comment:
what if the pod is not found? we should still proceed to try to delete

Reply:
Done in a new commit. PTAL.